rpc/grpc/proto3/blockchain.proto

```diff
@@ -143,3 +143,12 @@ message BlockTxsResponse {
   repeated google.protobuf.ListValue  Txs = 3 [(gogoproto.customtype) = "github.com/gallactic/gallactic/txs.Envelope"];
 }
 
+message BlockchainInfoRequest{
+  string Blockwithin =1;
```

Comment:
Do we need this? Just general information about blockchain is fair enough.  I think this parameter brings complication.